### PR TITLE
Updated Documenation and Added Example Code.

### PR DESCRIPTION
**Pull request by muktheeswaran**
Made changes as requested on this PR https://github.com/scrapy-plugins/scrapy-zyte-api/pull/3
Sorry for these two PR.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -64,15 +70,28 @@ key to download a request using Zyte API. Full list of parameters is provided in
...
-                "javascript": True,
-                "echoData": {"something": True}
-            }
-        }
-    )
+   import scrapy
+
+   class TestSpider(scrapy.Spider):
+       name = 'test'
+       start_urls = ['http://books.toscrape.com/']
```
Comment (BurnzZ) [inline README.rst:77]: We can remove this since `start_urls` isn't being used since `start_requests()` was overridden below.

File: README.rst [hunk 2]
```diff
@@ -64,15 +70,28 @@ key to download a request using Zyte API. Full list of parameters is provided in
...
+                                            "echoData": {"something": True}
+                                        }
+                                    })
+
+       def parse(self, response, **kwargs):
+           yield{
+               'URL': response.url,
+               'status': response.status,
+               'HTML': response.body
+           }
```
Comment (BurnzZ) [inline README.rst:97]: Let's format this code according to [black's ](https://github.com/psf/black) standards to fix the indentions and spacings.

File: README.rst [hunk 3]
```diff
@@ -64,15 +70,28 @@ key to download a request using Zyte API. Full list of parameters is provided in
...
+                                        "zyte_api": {
+                                            "browserHtml": True,
+                                            # You can set any GEOLocation region you want.
+                                            "geolocation": "US",
+                                            "javascript": True,
+                                            "echoData": {"something": True}
+                                        }
+                                    })
+
+       def parse(self, response, **kwargs):
```
Comment (BurnzZ) [inline README.rst:92]: `**kwargs` can be removed here.

**Review by BurnzZ — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (muktheeswaran) [reply README.rst]: Yes, I removed it now.

File: README.rst (hunk 2 above)
Comment (muktheeswaran) [reply README.rst]: Yes, used the black's standard for formatting now.

image

File: README.rst (hunk 3 above)
Comment (muktheeswaran) [reply README.rst]: Yup, removed that as well. and added a new Commit.

**Review by BurnzZ — APPROVED** [review]
Looking good now! Thanks @muktheeswaran! Congrats on your first GH PR! 🎉

**Pull request merged**